src/components/app/TabBar.js

```diff
@@ -51,6 +52,21 @@ class TabBar extends PureComponent {
   }
 }
 
+type Props = {
+  className?: string,
+  tabs: Tab[],
+  selectedTabName: string,
+  tabOrder: number[],
+  onSelectTab: string => mixed,
+  onChangeTabOrder: (number[]) => any,
+  +_mouseDownListener?: () => mixed,
```

Comment:
`onSelectTab` and `_mouseDownListener` should have been `void`, you're right.

As for `onChangeTabOrder`, it was looking like it needed to be both `{}` & `void` due to conflicts with `ProfileViewer` & `Reorderable`.

I was able to make everything pass by changing `TabBar` & `ProfileViewer` to both have `{}` return type.

I'm not sure if I can understand this, especially what the callback returning something would do. Must be a React thing?